src/components/HistoryTable/HistoryTable.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { HistoryTableProvider } from "./HistoryTableContext";
+import HistoryTable from "./index";
+import { mainlineCommitData } from "./testData";
+
+export default {
+  title: "History Table",
+};
+
+export const Default = () => (
```

Comment:
Could we consider moving the History Table Icon stories into this file so that they're nested together?